documentation/assemblies/upgrading/assembly-upgrade-kafka-cluster-operator.adoc

```diff
@@ -7,7 +7,7 @@
 
 The steps to upgrade your Cluster Operator deployment to use Strimzi {ProductVersion} are described in this section.
 
-Follow this procedure if you deployed the Cluster Operator using the installation YAML files.
+Follow this procedure if you deployed the Cluster Operator using the installation YAML files rather than {OperatorHub}.
```

Comment:
What about helm?

Reply:
I think we'll need to open a new issue for Helm.

The Deploying and upgrading Strimzi guide should be updated to reflect deployment using Helm charts. The Upgrading Strimzi chapter needs to describe how to upgrade Strimzi if you deployed via Helm.

What do you think, @scholzj ?

Reply:
I think that Kafka upgrades are the same regardless of OLM or Helm. For the operator upgrade, I'm not sure we have anyone who knows how to do it with Helm :-/